bot/handlers/rooms.py

```diff
@@ -109,7 +109,7 @@ def cmd_dafi(update: Update, context: CallbackContext[dict, dict, dict]) -> None
         context.bot_data['room_queue'] = queue
 
     if action == RoomActions.ON:
-        if user.telegram_id in members:
```

Comment:
Ya que tenemos `members_ids` disponibles, mejor utilizamos eso: `user.telegram_id in members_ids`.

Reply:
De acuerdo, pensé en hacerlo pero opté por la opción que he puesto